src/audit-logs/interfaces/audit-log-export-options.interface.ts

```diff
@@ -1,5 +1,8 @@
 export interface AuditLogExportOptions {
+  actions?: string[];
```

Comment:
Seems like it would make more sense to have these grouped together at the end since they're all optional filters rather than sorting the fields alphabetically.